types/index.ts

```diff
@@ -13,3 +13,14 @@ export type SelectedShift = {
   id: number;
   shift: string;
 };
+
+export interface BeforeInstallPromptEvent extends Event {
```

Comment:
If this really needs to be an interface, change the name to follow guidelines (`IBeforeInstallPromptEvent`).